Zend/tests/nullsafe_method_call/014.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+Test nullsafe in binary op
+--FILE--
+<?php
+
+$foo = null;
+
+var_dump($foo?->foo() + $foo?->bar());
+var_dump(($foo?->foo() + $foo?->bar())?->baz());
```

Comment:
There's probably some more variations worth testing, to illustrate where short circuiting does *not* happen. E.g. `var_dump($foo?->foo() + $foo->bar());` should not short-circuit the second call.